spotlight-client/src/charts/BarChartTrellis.tsx

```diff
@@ -81,6 +81,7 @@ export function BarChartTrellis({
   return (
     <MeasureWidth>
       {({ measureRef, width }) => (
+        //note for PR
         <ChartWrapper ref={measureRef}>
```

Comment:
throwing this one to @macfarlandian, thoughts?

Reply:
yeah we should be able to do that!

Reply:
https://github.com/Recidiviz/web-libraries/issues/93 ... when this is merged and a new package version is published you should be unblocked on this

Reply:
Hello Ilya! Pleasure to virtually meet you! Design System v3.3.0's ChartWrapper can now accept and forward a `ref`. 

https://github.com/Recidiviz/web-libraries/issues/93 has been closed by https://github.com/Recidiviz/web-libraries/issues/94

I look forward to working with & learning from you! :pray:

Reply:
@mxosman Thank you! But now this component throws ts-error. I assume something wrong with children. Error: 

`No overload matches this call.
  Overload 1 of 2, '(props: Omit<Omit<ChartWrapperProps & RefAttributes<HTMLDivElement>, never> & Partial<Pick<ChartWrapperProps & RefAttributes<...>, never>>, "theme"> & { ...; } & { ...; }): ReactElement<...>', gave the following error.
    Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & Omit<Omit<ChartWrapperProps & RefAttributes<HTMLDivElement>, never> & Partial<...>, "theme"> & { ...; } & { ...; }'.
  Overload 2 of 2, '(props: StyledComponentPropsWithAs<ForwardRefExoticComponent<ChartWrapperProps & RefAttributes<HTMLDivElement>>, any, {}, never>): ReactElement<...>', gave the following error.
    Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & Omit<Omit<ChartWrapperProps & RefAttributes<HTMLDivElement>, never> & Partial<...>, "theme"> & { ...; } & { ...; }'.`

Reply:
@macfarlandian, also, after bumping design-system to v3, some strange errors appeared in tests, and idk honestly why they're there. Can you look at this? And is it even worth it? It seems like we'll need custom styling for `ChartWrapper` anyway, so we'll just wrap `ChartWrapper` from design-system in existing component.

Reply:
without seeing your branch I can't be sure but I am guessing those test failures are caused by the breaking changes in design system 3.x, which now requires a styled-components `ThemeProvider` (as seen [here](https://github.com/Recidiviz/pulse-dashboard/blob/main/src/App.js#L82), for example). You can probably address this by adding this to the rendering functions in `testUtils.tsx` and possibly adding a new function also if you want to render something for test without a store or a router that requires a theme provider.

Reply:
Sorry to your other point, I wouldn't think you really need any more custom styling than you do now? The `ChartWrapper` in the design system is nearly identical to the one in this application, it was basically just copied out of here with minor modifications. The handful of charts that extend it with additional styles do so because those styles are either not needed for most other charts or might actually conflict with them. If that's not the case, we can reassess whether this is worthwhile, but if there are some minor differences that don't actually look bad then it shouldn't be necessary to override them locally. Once you have an updated branch I can take a closer look

Reply:
> @mxosman Thank you! But now this component throws ts-error. I assume something wrong with children. Error:
> 
> `No overload matches this call. Overload 1 of 2, '(props: Omit<Omit<ChartWrapperProps & RefAttributes<HTMLDivElement>, never> & Partial<Pick<ChartWrapperProps & RefAttributes<...>, never>>, "theme"> & { ...; } & { ...; }): ReactElement<...>', gave the following error. Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & Omit<Omit<ChartWrapperProps & RefAttributes<HTMLDivElement>, never> & Partial<...>, "theme"> & { ...; } & { ...; }'. Overload 2 of 2, '(props: StyledComponentPropsWithAs<ForwardRefExoticComponent<ChartWrapperProps & RefAttributes<HTMLDivElement>>, any, {}, never>): ReactElement<...>', gave the following error. Type '{ children: Element; }' has no properties in common with type 'IntrinsicAttributes & Omit<Omit<ChartWrapperProps & RefAttributes<HTMLDivElement>, never> & Partial<...>, "theme"> & { ...; } & { ...; }'.`

Acknowledged - will let you know as soon as this is sorted out.

Reply:
Hello @nasaownsky - Design System v.3.3.1 fixes the TS issue. Let me know if you run into any more issues with this. Thank you for your patience.